Makefile

```diff
@@ -347,8 +347,8 @@ clean:
 ## -- Targets for uploading code coverage reports to Codecov.io--
 
 .PHONY: upload-codecov-report
-# Uploads the test coverage reports to codecov.io.
-# DO NOT USE LOCALLY: must only be called by OpenShift CI when processing new PR and when a PR is merged!
+## Uploads the test coverage reports to codecov.io.
+## DO NOT USE LOCALLY: must only be called by OpenShift CI when processing new PR and when a PR is merged!
```

Comment:
Why it can't run locally?

Reply:
It was mentioned already in master. Just added an extra # so that it shows up on `make help`